scripts/install

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+#=================================================
+# GENERIC START
+#=================================================
+# IMPORT GENERIC HELPERS
+#=================================================
+
+source _common.sh
+source /usr/share/yunohost/helpers
+
+# Install parameters are automatically saved as settings
+#
+# Settings are automatically loaded as bash variables
+# in every app script context, therefore typically these will exist:
+# - $domain
+# - $path
+# - $language
+# ... etc
+#
+# Resources defined in the manifest are provisioned prior to this script
+# and corresponding settings are also available, such as:
+# - $install_dir
+# - $port
+# - $db_name
+# ...
+
+#=================================================
+# APP "BUILD" (DEPLOYING SOURCES, VENV, COMPILING ETC)
+#=================================================
+# DOWNLOAD, CHECK AND UNPACK SOURCE
+#=================================================
+ynh_script_progression --message="Setting up source files..." --weight=1
+
+### `ynh_setup_source` is used to install an app from a zip or tar.gz file,
+### downloaded from an upstream source, like a git repository.
+### `ynh_setup_source` use the file conf/app.src
+
+# Download, check integrity, uncompress and patch the source from app.src
+ynh_setup_source --dest_dir="$install_dir"
+
+# $install_dir will automatically be initialized with some decent 
+# permission by default ... however, you may need to recursively reapply
+# ownership to all files such as after the ynh_setup_source step
+chown -R $app:www-data "$install_dir"
+
+
+#=================================================
+# INSTALL PYTHON DEPENDENCIES
+#=================================================
+ynh_script_progression --message="Installing python dependencies..." --weight=1
+
+pushd $install_dir
+  python3 -m venv venv
+  . venv/bin/activate
+  venv/bin/pip3 install --upgrade pip
+  venv/bin/pip3 install vosk
+  venv/bin/pip3 install -r requirements.txt # should only be flask for now, but just in case
+popd
+
+# Install DPM in a virtual environement
+pushd $install_dir/punctuation-server
+  python3 -m venv venv
+  . venv/bin/activate
+  venv/bin/pip3 install -r requirements.txt # should only be flask for now, but just in case
+  DMP_SERVER_INTERFACE=0.0.0.0 DMP_SERVER_PORT=$port_dmp_server_port python3 server.py # TODO : Replace by a systemd service
```

Comment:
I think this could be the cause for the time out.

Reply:
You mean it could just run forever ?
I need to convert that to a systemd service, but I don't know yet how to do it and I wanted to keep it simple… will comment it :)